src/QueryEditor.tsx

```diff
@@ -63,9 +63,10 @@ export class QueryEditor extends PureComponent<Props> {
             portalOrigin="lightstep"
             placeholder="Enter a PromQL query"
```

Comment:
Would you mind updating the placeholder text so users know to actually use Shift+Enter? We could use something similar to the Prometheus placeholder: "Enter a PromQL query (run with Shift+Enter)"

Reply:
oh yes absolutely! that is very wise